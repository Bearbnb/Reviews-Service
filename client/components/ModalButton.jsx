import React from 'react';
import ReactDOM from 'react-dom';
import PropTypes from 'prop-types';
import styles from '../../styles/FlagModal.css';


const ModalButton = ({ header, content, changeRadio}) => (
    <div className={styles.ButtonWrapper}>
     <div className ={styles.ModalButtonHeader}>
        {header}
     </div> 
     <div className ={styles.ModalButtonContent}>
         {content}
        <input type="radio" className={styles.ModalRadio} onClick={() => changeRadio()}/>
     </div>
    </div>
);

export default ModalButton;
