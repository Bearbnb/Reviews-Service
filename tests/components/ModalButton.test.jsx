import { shallow, mount, render } from 'enzyme';
import toJson from 'enzyme-to-json';
import React from 'react';
import ModalButton from '../../client/components/ModalButton';

describe('<ModalButton />', () => {
  it('should render properly', () => {
    const wrapper = shallow(<ModalButton
      header=""
      content=""
      changeRadio={jest.fn()}
    />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
