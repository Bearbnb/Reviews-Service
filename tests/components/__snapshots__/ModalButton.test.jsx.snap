// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ModalButton /> should render properly 1`] = `
<div
  className="ButtonWrapper"
>
  <div
    className="ModalButtonHeader"
  />
  <div
    className="ModalButtonContent"
  >
    <input
      className="ModalRadio"
      onClick={[Function]}
      type="radio"
    />
  </div>
</div>
`;
